src/test/java/net/datafaker/helpers/IdNumberPatterns.java

```diff
@@ -10,5 +10,6 @@ public class IdNumberPatterns {
     public static final Pattern FRENCH = Pattern.compile("[12]\\d{2}(1[0-2]|0[1-9])\\d[0-9a-zA-Z]\\d{8}");
     public static final Pattern ITALIAN = Pattern.compile("[A-Z]{6}\\d{2}[ABCDEHLMPRST]\\d{2}[\\dA-Z]{5}");
     public static final Pattern POLISH = Pattern.compile("\\d{11}");
+    public static final Pattern HUNGARIAN = Pattern.compile("\\d{10}");
```

Comment:
Should we specify that the first digit should be in the set 1-4?

Reply:
Good point. Maybe.
Other numbers are theoretically possible, but we don't use them because it's legacy.

Reply:
done.